lib/group_email_credentials_check.rb

```diff
@@ -7,6 +7,9 @@
 # problem checks, and if any credentials have issues they will show up on
 # the admin dashboard as a high priority issue.
 class GroupEmailCredentialsCheck
+  def self.retries = 2
+  def self.retry_wait = 30.seconds
```

Comment:
First time I use "endless" methods. Thought this was a pretty nice use case.

Reply:
Oh this is interesting!! Kind of reminds me of `const myFunc = () => 2` in JS

Reply:
This would be quite nice for my hashtag and bookmarkable stuff

https://github.com/discourse/discourse/blob/4346512f88ebf3fa4aa0cd6a68d98781fe70423d/app/services/post_bookmarkable.rb#L3-L16